Examples/src/components/Examples/Charts2D/ChartAnnotations/EditableAnnotations/GENERATED_SRC.ts

```diff
@@ -100,8 +100,8 @@ export const drawExample = async () => {
             isEditable: true
         }),
         new TextAnnotation({
-            x1: 4,
-            y1: 2,
```

Comment:
what are these hardcoded values?

Reply:
I changed it for a better view on mobile devices